### PR TITLE
New login screen

### PR DESCRIPTION
**Pull request by dohseven**
Adapt the konnector to the new login screen

### CODE REVIEW

File: src/keypad.js [hunk 1]
```diff
@@ -121,22 +6,21 @@ function getPassword($, password) {
...
-    const limits = getImageLimits(pngImg, { R: 255, G: 255, B: 255 })
+    // Convert base64 encoded image to SVG text description
+    const svgImg = Buffer.from(b64pic, 'base64').toString('utf-8')
 
-    // Find the digit from the limits and associate it to its key
+    // Extract digit from SVG text description
     const keypad = {
       key: $button.attr('data-matrix-key'),
-      digit: getDigit(limits)
+      digit: svgImg.match(/ id="(\d)"/)[1]
```
Comment (orandin) [inline src/keypad.js:23]: `BaseKonnector: critical : Cannot read property '1' of null`

Now, SVG no longer contains the "id" attribute ... Boursorama has updated these images :'(

**Comment by orandin** [comment]
@dohseven : I found another way with base64 :)
It may be a temporary solution, but I haven't found any other way yet.

**Comment by dohseven** [comment]
@orandin 
> Can you check my commit and tell me if everything works on your side?

Works for me!

But you're right, I'm afraid the SVG content will slightly change in the future, breaking the current implementation.

[Weboob](https://git.weboob.org/weboob/weboob/merge_requests/473) tries to find the closest match in order to avoid this, but I'm not sure if this is the best solution and even how to implement it in NodeJS...

**Comment by orandin** [comment]
Wait and see if SVG content will be updated :)

**Pull request merged**